lib/health_check/middleware_health_check.rb

```diff
@@ -0,0 +1,42 @@
+module HealthCheck
+  class MiddlewareHealthcheck
+
+    def initialize(app)
+      @app = app
+    end
+
+    def call(env)
+      uri = env['PATH_INFO'.freeze]
+      if uri.include? HealthCheck.uri
+        response_type = uri.sub! '/' + HealthCheck.uri + '.' , ''
+        response_method = 'response_' + response_type.to_s
+        checks = 'standard'
+        begin
+          errors = HealthCheck::Utils.process_checks(checks)
+        rescue Exception => e
```

Comment:
http://stackoverflow.com/questions/10048173/why-is-it-bad-style-to-rescue-exception-e-in-ruby
maybe just rescue